inc/buddyboss.php

```diff
@@ -329,3 +329,43 @@ function nightingale_theme_following_count( $user_id = false ) {
 	}
 }
 
+
+if ( ! function_exists( 'buddyboss_pagination' ) ) {
+
+	/**
+	 * Custom Pagination
+	 */
+	function buddyboss_pagination() {
+		global $paged, $wp_query;
+
+		$max_page = 0;
+
+		if ( ! $max_page ) {
```

Comment:
Why to check this? it is 0 anyway defined line above which resolves to FALSE... Consider removing both the `0` declaration  and if statement also Should it be ```$max_pages``` not max_page ?